README.md

```diff
@@ -1,2 +1,186 @@
 # trampss-redux-data-store
```

Comment:
The librairy name doesn't speak itself... and is very long.
Does it be better to call it with `factory` in the lib name, like : `redux-factory` or `trampss-redux-factory` ?